assignments/boolean-birds/code.js

```diff
@@ -16,3 +16,39 @@
  *   - Globby Birds eat fish, mice, and nuts.
  *
  */
+const isFlobbyBird = (red, spotted) => {
+  return red && spotted
+}
+const isBloggyBird = (red, spotted) => {
+  return red && ! spotted
+}
+const isFlibbleBird = (red, spotted) => {
+  return !red && spotted
+}
+const isGlobbyBird = (red, spotted) => {
+  return !red && ! spotted
+}
+const eatsWorms = (FlobbyBirds, BloggyBirds, FlibbleBirds, GlobbyBirds) => {
+  return FlobbyBirds || FlibbleBirds || BloggyBirds &&! GlobbyBirds
+}
+const eatsNuts = (red, spotted) => {
+  return red ||! spotted
+}
+const eatsFish = (red, spotted) => {
+  return spotted || !red
+}
+const eatsMice = (red, spotted) => {
+  return !spotted || !red
+}
+const isRed = (redburd) => {
+  return redburd=== 'Flobby' ||redburd=== 'Bloggy'
```

Comment:
Hit the auto formatter on this code. There should be spaces around the `===` and `||` operators.

```suggestion
  return redburd === 'Flobby' || redburd === 'Bloggy'
```